lms/djangoapps/learner_home/test_views.py

```diff
@@ -184,7 +186,10 @@ def mock_get_filtered_course_entitlements(
             course_entitlement_available_sessions,
             unfulfilled_entitlement_pseudo_sessions,
         )
-        with patch('lms.djangoapps.learner_home.views.get_filtered_course_entitlements', return_value=return_value):
```

Comment:
Your autolinter is too aggressive with the line breaks. What's it set at, 100? Our style checkers are at 120.